### DISCUSSION TITLE
Ongoing work on the Shops page 🚀

### DISCUSSION THREAD

**Discussion by tamalCodes** [karmacircleCommunity/KarmaCircle#730] [General]
**Currently, we are adding** 

- an API for the admins to push products in the database in #728 , and after that, 
- an API for fetching the products in #700  - Thanks to @Nitish-hack for the Huge backend progress 🔥

Also @KGoutham7214 will be helping us in revamping the whole products card in #729 

All I need after that is to make sure in the FRONTEND that the APIs made by @Nitish-hack are properly integrated too.

> **Note**
> I am jotting down this Discussion in order to help all of us collaborate !

**Comment by tamalCodes** [comment]
Update: Work going on `adding to cart` logic will be worked by @PatilHarshh in #708

**Comment by PatilHarshh** [comment]
sure @tamalCodes, Thanks for assigning :)

**Comment by YuvarajSingh-0** [comment]
the card UI seems outdated, don't you think? @tamalCodes

**Comment by tamalCodes** [comment]
The designing has started, so for now I'm thinking to put all design related issues on halt.
